### ISSUE TITLE
Form variables don't display in success message

### ISSUE THREAD

**Issue by saltymouse** [verbb/formie#352]
**Description**
I've setup a simple contact form that displays inside a pop-up modal using the default `{{ craft.formie.renderForm('momentForm') }}` render method. Inside the form's "Behavior" settings, I'm using the "First Name" variable from the form in the "Submission Message", however, this fails to appear on the front end after sending.

![formie-behavior-submit-message-var](https://user-images.githubusercontent.com/7547649/108414539-35da2080-71e1-11eb-8e8b-df332c0dd35a.png)

![formie-behavior-submit-message-result](https://user-images.githubusercontent.com/7547649/108414558-3bd00180-71e1-11eb-8ea2-b90b1782e2fc.png)

Trying other variables from the form has the same result as "First Name", too.

### Form settings
- Multi-page form: **No**
- Submission Method: **Ajax**
- Client-side Validation: **Yes (Formie's)**
- Custom Form Templates: **No**

**Additional info**
- Plugin version: **1.3.17**
- Craft version: **3.6.6**
- Multi-site: **No**

**Comment by engram-design** [comment]
Should be fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.17",
  "...": "..."
}
```

Then run `composer update`.

**Comment by saltymouse** [comment]
Just tried `"verbb/formie": "dev-craft-3 as 1.3.17"` and can confirm fixed now! Thanks for quick fix—cheers!

**Comment by engram-design** [comment]
Fixed in [1.3.18](https://github.com/verbb/formie/releases/tag/1.3.18)
